.github/workflows/ci.yml

```
@@ -81,19 +87,26 @@ jobs:
 
           make test-ui-start-pod
 
+      - name: Start Grafana
+        run: make GRAFANA_IMAGE=docker.io/grafana/${{ matrix.grafana_image }} test-ui-start-grafana-build
+
+      - name: Wait for Grafana to be ready
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2034:warning:1:1: i appears unused. Verify use (or export if used externally)

(shellcheck)